example/declarative.dart

```diff
@@ -1,29 +1,31 @@
 // Copyright 2013 Google. All rights reserved. Use of this source code is
 // governed by a BSD-style license that can be found in the LICENSE file.
 
-library grinder_example_1;
+library grinder.example.declarative;
 
 import 'package:grinder/grinder.dart';
 
-void main([List<String> args]) {
-  task('init', init);
-  task('compile', compile, ['init']);
-  task('deploy', deploy, ['compile']);
-  task('default', null, ['deploy']);
-
+void main(List<String> args) {
```

Comment:
That's what I was going for with `export 'package:grinder/main.dart';`.  I also considered what you did above i.e. calling `addAnnotatedTasks()` within `startGrinder()`, but if we do want to allow people to opt out of annotation scanning, then we will might still want to allow them to use `startGrinder`.  So another idea is to add this method to grinder:

```dart
void grind(List<String> args) {
  addAnnotatedTasks();
  startGrinder();
}
```

Then the 90% use case can be:

```dart
main(args) => grind(args);
```
or even:

```dart
const main = grind;
```

I definitely agree having a single entry-point for most use cases is nice, then we can change what it does without it being a breaking change for most.

Just let me know which option you like best, and I can implement it.